db/migrations/20250420234337_moderation_tables.sql

```diff
@@ -0,0 +1,77 @@
+-- +goose Up
+-- +goose StatementBegin
+
+CREATE TABLE report_reasons (
+	report_reason_id bigint PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
```

Comment:
tiny nit, but maybe we could use `integer` or even `smallint` for this column?

Reply:
I noticed that every other table uses `bigint` for the PK, without exception, so I chose `bigint` to be consistent with that. Thoughts?

Reply:
yep fine 👍